Jenkinsfile

```diff
@@ -18,36 +17,48 @@ def localSrcFolder = '.'
 def lcovFile = './test-output/lcov.info'
 def timeoutInMinutes = 5
 
-
 node {
   checkout scm
   try {
+    stage('Set GitHub status as pending'){
+      defraUtils.setGithubStatusPending()
+    }
     stage('Set branch, PR, and containerTag variables') {
       (pr, containerTag, mergedPrNo) = defraUtils.getVariables(repoName)
-      defraUtils.setGithubStatusPending()
     }
     stage('Helm lint') {
-      defraUtils.lintHelm(imageName)
+      defraUtils.lintHelm(repoName)
     }
     stage('Build test image') {
-      defraUtils.buildTestImage(imageName, BUILD_NUMBER)
+      defraUtils.buildTestImage(repoName, BUILD_NUMBER)
     }
     stage('Run tests') {
-      defraUtils.runTests(imageName, BUILD_NUMBER)
+      defraUtils.runTests(repoName, BUILD_NUMBER)
     }
     stage('Push container image') {
-      defraUtils.buildAndPushContainerImage(regCredsId, registry, imageName, containerTag)
-    }
+      defraUtils.buildAndPushContainerImage(regCredsId, registry, repoName, containerTag)
+    } 
+
     if (pr == '') {
+      stage('Verify version incremented') {
```

Comment:
This needs to move to the else statement.  We want this to run on PR only so PR would not equal '' in that case.

Reply:
Code moved inside else statement.